recipes-aos/aos-updatemanager/files/aos_updatemanager.cfg

```diff
@@ -1,6 +1,6 @@
 {
     "ID": "um",
-    "ServerUrl": "aossm:8091",
+    "ServerUrl": "aoscm:8091",
     "CACert": "/etc/ssl/certs/rootCA.pem",
     "CertStorage": "/var/aos/crypt/um/",
```

Comment:
Also, add dep in UM service file to start after CM.

Reply:
added